syntax = "proto3";

package million.ss.service;

message ServiceStart {
    
}

message ServiceStop {
    
}

message SessionTimeout {
    uint64 timeout_id = 1;
}

message ServiceUnlock {
    
}